pdfplumber/container.py

```diff
@@ -47,6 +47,22 @@ def images(self):
     def chars(self):
         return self.objects.get("char", [])
 
+    @property
+    def textboxverticals(self):
+        return self.objects.get("textboxvertical", [])
```

Comment:
The name of the method is plural but the name of the key is singular. Shouldn't both be same?

Reply:
I also find this unsatisfying, but it parallels the approach of the preexisting code — e.g., `page.objects["char"]` + `page.chars`. I'm open to altering this approach in a future major version. Or perhaps I'm overlooking a more elegant solution? Thoughts?

Reply:
I don't have any answer at the moment but yeah, something to consider changing in a next major version.